esp-hal/src/uart.rs

```diff
@@ -507,6 +526,7 @@ impl Default for Config {
             clock_source: Default::default(),
             rx_fifo_full_threshold: UART_FULL_THRESH_DEFAULT,
             rx_timeout: Some(UART_TOUT_THRESH_DEFAULT),
+            tx_idle_num: 256,
```

Comment:
Why not default to 0?

Reply:
256 is the default shown in the technical specification.